src/index.ts

```diff
@@ -3,7 +3,7 @@ import parse from 'inline-style-parser';
 
 export { Declaration };
 
-interface StyleObject {
+export interface StyleObject {
```

Comment:
Question: instead of exporting the interface, could you do this?

```ts
import styleToObject from 'style-to-object';

type StyleObject = NonNullable<ReturnType<typeof styleToObject>>;

```

Reply:
Yes, it may be a solution.  
Now, is redefining a type more readable and easy than just importing it from source? Like:
```ts
import type { StyleObject } from 'style-to-object';
```
For me, the import is clearer, more efficient (the type is already in memory, no need to infer it again), and remains consistent with usual practices.